### PR TITLE
Refactor .inline-logos to .p-image-grid 

### LINKED ISSUE #403: Audit inline logos
## Context

- Sanity check source code.
- Remove examples of specific logs and replace with generic asset
- Should possible be renamed to block grid or similar to remove explicit use case of logos, could be used for any list of images

## Audit checklist

- [ ] Browser testing
- [ ] Accessibility testing
- [ ] Sass linting
- [ ] Update documentation
- [ ] Convert sizing units to rems
- [ ] Namespace styling to `vf-`

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Simplified this pattern and removed superfluous styling
- Remove focus on logos to more generic images for multi-use
- Updated naming
- Updated docs

## QA

- Sanity check code
- Sync with vf.io and test pattern

## Issues

Fixes #403

### CODE REVIEW

File: docs/patterns/_image-grid.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+collection: patterns
+title: image grid
+---
+
+This image grid can be used to showcase a group of related image such as logos.
+
+```html
+<div class="p-image-grid">
```
Comment (anthonydillon) [inline docs/patterns/_image-grid.md]: I think we should demo this as a list.

File: scss/_patterns_image-grid.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Image grid to showcase group of images
+@mixin vf-p-p-image-grid {
+
+  .p-p-image-grid {
```
Comment (anthonydillon) [inline scss/_patterns_image-grid.scss]: `p-p-`? Is this the pattern to end all patterns?

Comment (anthonydillon) [inline scss/_patterns_image-grid.scss]: `p-p-`? Is this the pattern to end all patterns?

File: scss/_patterns_image-grid.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    @extend .clearfix;
+    text-align: center;
+    display: block;
+
+    &__img {
```
Comment (anthonydillon) [inline scss/_patterns_image-grid.scss]: We need to control the max-width and height here. As rarely have the correct sized images. Don't SVGs are used and they can be large.

File: scss/_patterns_strip.scss [hunk 4]
```diff
@@ -1,15 +1,17 @@
 // Full width strip pattern
 @mixin vf-p-strip {
 
-  width: 100%;
-  clear: both;
-  background-color: $color-light;
+  .strip {
```
Comment (anthonydillon) [inline scss/_patterns_strip.scss]: Looks like this change is also included in the breadcrumbs PR. With a few comments, please resolve them before landing this branch.

**Comment by anthonydillon** [comment]
Thanks Barry, few comments on the code.

File: scss/_patterns_image-grid.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      display: inline-block;
+      text-align: center;
```
Comment (barrymcgee) [inline scss/_patterns_image-grid.scss]: Yep, removing it breaks the pattern...

File: scss/_patterns_image-grid.scss (hunk 3 above)
Comment (barrymcgee) [inline scss/_patterns_image-grid.scss]: I'd prefer to leave this light touch as responsibility for sourcing and using correctly sized images should be on the author.. by adding height/width stipulations, we leave the pattern more brittle and also open the possibility of people using huge images without realising...

**Pull request merged**